.github/workflows/publish-store.yml

```
@@ -128,3 +136,18 @@ jobs:
               changelog:"$CHANGELOG" \
               metadata_path:"$METADATA_PATH" \
               submit_release:"$SUBMIT_RELEASE"
+
+        - name: Enable Publish Github Release Workflow
+          if: ${{ steps.publish.conclusion == 'success' }}
+          env:
+            PRODUCT: ${{ inputs.product }}
+          run: |
+            if ${{ inputs.dry-run }} ; then
+              gh workflow view publish-github-release.yml
+              exit 0
+            fi
+            if $PRODUCT == "Password Manager" ; then
+              gh workflow enable publish-github-release-pwm.yml
```

Along the same lines as above

```suggestion
              gh workflow enable publish-github-release-pm.yml
```